docs/pages/reference/helm-reference/includes/zz_generated.teleport-kube-agent.mdx

```diff
@@ -0,0 +1,1627 @@
+
+This value serves the same purpose as [`authToken`](#authToken) but supports
+all join methods. When set, it takes precedence over `authToken`.
+Its usage should be preferred.
+
+### `joinParams.method`
+
+| Type | Default |
+|------|---------|
+| `string` | `"token"` |
+
+`joinParams.method` controls which join method will be used by the
+instance join the Teleport cluster.
+
+See [the join method reference](../../join-methods.mdx) for the list of possible
+values, the implications of each join method, and guides to setup each method.
+
+Common join-methods for the `teleport-kube-agent` are:
+- `token`: the most basic one, with regular ephemeral secret tokens
+- `kubernetes`: either the `in-cluster` variant (if the agent runs in the
+  same Kubernetes cluster as the `teleport-cluster` chart) or the `JWKS`
+  variant (works in every Kubernetes cluster, regardless of the Teleport Auth
+  Service location).
+
+### `joinParams.tokenName`
+
+| Type | Default |
+|------|---------|
+| `string` | `""` |
+
+`joinParams.tokenName` controls which token is used by the agent to
+join the Teleport cluster.
+
+When `joinParams.method` is [a delegated join method](../../join-methods.mdx#delegated-join-methods),
+the value is not sensitive.
+
+When `joinParams.method` is `token` (by default), `joinParams.tokenName`
+contains the secret token itself. In this case, the value is sensitive and
+is automatically stored in a Kubernetes Secret instead of being directly
+included in the agent's configuration.
+
+If method is `token`, `joinParams.tokenName` can be empty if the token
+is provided through an existing Kubernetes Secret, see
+[`joinTokenSecret`](#joinTokenSecret) for more details and instructions.
+
+## `kubeClusterName`
+
+| Type | Default |
+|------|---------|
+| `string` | `""` |
+
+`kubeClusterName` sets the name used for the Kubernetes cluster proxied by
+the Teleport agent. This name will be shown to Teleport users connecting to
+the cluster.
+
+This setting is required if the chart `roles` contains `kube`.
+
+## `apps`
+
+| Type | Default |
+|------|---------|
+| `list` | `[]` |
+
+`apps` is a static list of applications that should be proxied by
+the agent. See [the Teleport Application access documentation](../../../application-access/introduction.mdx)
+for more details.
+
+Proxied applications can be defined statically (through this value) or dynamically
+(through the [`appResources`](#appResources) value).
+One of `apps` and `appResources` is required if the chart `roles` contains `app`.
+
+You can specify multiple apps by adding additional list elements.
+For example:
+
+```yaml
+apps:
+  - name: grafana
+    uri: http://localhost:3000
+    labels:
+      purpose: monitoring
+  - name: jenkins
+    uri: http://jenkins:8080
+    labels:
+      purpose: ci
+```
+
+<Admonition type="tip" title="Supported values">
+  You can see a list of all the supported[values which can be used in a Teleport
```

Comment:
-> "You can see a list of all the supported values that can be used in a Teleport Application Service configuration in the [Application Service Configuration Reference]"

- Describes the link destination
- The config is for the Application Service